src/iop/colorin.c

```diff
@@ -1938,6 +1941,27 @@ void reload_defaults(dt_iop_module_t *module)
       }
     }
 #endif
+#ifdef HAVE_LIBHEIF
+    else if(!strcmp(ext, "heif") || !strcmp(ext, "heic") || !strcmp(ext, "hif"))
+    {
+      struct heif_color_profile cp = {
+          .type = DT_COLORSPACE_NONE,
+      };
+
+      dt_imageio_heif_read_color_profile(filename, &cp);    
```

Comment:
See also https://github.com/darktable-org/darktable/pull/8807

Reply:
This is fixed.